common/src/services/search.service.ts

```diff
@@ -294,12 +295,13 @@ export class SearchService implements SearchServiceAbstraction {
     const checkFields = fields.every((i: any) => searchableFields.includes(i));
 
     if (checkFields) {
-      return token
-        .toString()
-        .normalize("NFD")
-        .replace(/[\u0300-\u036f]/g, "");
+      return this.normalizeSearchQuery(token.toString());
     }
 
     return token;
   }
+
+  private normalizeSearchQuery(query: string): string {
+    return query?.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Comment:
On second thoughts - please add a comment here explaining what this does, regexes with unicode ranges are not very self-explanatory. (e.g. "Normalize unicode string to its decomposed form and then remove all accents. This unicode range is equivalent to the Diacritic unicode property escape, but is backwards-compatible with older browsers.")